MapboxNavigation/FeedbackCollectionViewCell.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+import UIKit
+
+
+class FeedbackCollectionViewCell: UICollectionViewCell {
+    static let defaultIdentifier = "MapboxFeedbackCell"
```

Comment:
`defaultCellIdentifier` ?

Reply:
I would argue that this would be redundant name-spacing, since we already know it's context (that it's a cells' identifier) at the call-site: `FeedbackCollectionViewCell.defaultIdentifier`

Reply:
Yes, let's keep as is.